A two sentence fix/clarification for the access API guide (#11471)
IQSS/dataverse: doc/sphinx-guides/source/api/native-api.rst

.. note:: In addition to the files and folders in the dataset, the command line above will also save the directory index of each folder, in a separate folder "dirindex".

> .. note:: The recipe above does NOT work if the Dataverse instance serves the files via direct download (i.e., when instead of streaming the content, the download API issues redirect links to the files stored on S3). Unfortunately, this describes **every** file served by the Dataverse instance at Harvard, among others. With direct download, saving an entire dataset while preserving its folders structure is possible, but requires some extra scripting.  

Reviewer: ```suggestion
.. note:: The recipe above does NOT work if the Dataverse instance serves the files via direct download (i.e., when instead of streaming the content, the download API issues redirect links to the files stored on S3). Unfortunately, this describes **every** file served many instances of Dataverse. With direct download, saving an entire dataset while preserving its folders structure is possible, but requires some extra scripting.  
```

Just a suggestion to remove Harvard. Perhaps we could write about this at https://support.dataverse.harvard.edu ?

Author: how about "by many popular Dataverse instances"?

Reviewer: Sure!
